packages/eslint-config-base/package.json

```diff
@@ -24,14 +24,17 @@
   },
   "dependencies": {
     "@rushstack/eslint-patch": "1.10.3",
-    "eslint-config-airbnb-base": "15.0.0",
+    "confusing-browser-globals": "1.0.11",
     "eslint-import-resolver-typescript": "3.6.1",
     "eslint-plugin-import": "npm:eslint-plugin-i@2.29.1",
     "eslint-plugin-simple-import-sort": "12.1.0",
-    "eslint-plugin-unicorn": "54.0.0"
+    "eslint-plugin-unicorn": "54.0.0",
+    "semver": "7.6.2"
   },
   "devDependencies": {
-    "@rightcapital/tsconfig": "workspace:*"
+    "@rightcapital/tsconfig": "workspace:*",
+    "@types/confusing-browser-globals": "1.0.3",
+    "@types/semver": "7.5.8"
```

Comment:
Consider moving these two dependencies to `dependencies` section?

Reply:
We are not reexporting these types; It is not very necessary to do so IMO

Reply:
cc @liby What do you think ? 👀

Reply:
Moving type definition packages to `dependencies` can cause unnecessary bloat and [version conflicts](https://github.com/DefinitelyTyped/DefinitelyTyped/issues/55519) in production. It's generally better to keep them in `devDependencies` to avoid these issues, unless runtime type definitions are specifically needed. This keeps the dependency tree cleaner and more manageable.

Reply:
After discussion, moving `@types/*` to `dependencies` does not have any obvious issues. This is more like a style preference issue:

1. I prefer to put type packages that are not exported in `devDependencies`.

2. I prefer to put the type package and its package together.

This reminds me of discussions about indentation, and if we want to choose a style preference, we can only vote within the entire team 🥹